<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="45" started-at="2020-05-07T19:44:35Z" finished-at="2020-05-07T19:44:35Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="45" started-at="2020-05-07T19:44:35Z" finished-at="2020-05-07T19:44:35Z">
      <class name="testing.TC_001">
        <test-method status="PASS" signature="enterURL()[pri:0, instance:testing.TC_001@7c0e2abd]" name="enterURL" is-config="true" duration-ms="7" started-at="2020-05-08T01:14:35Z" finished-at="2020-05-08T01:14:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- enterURL -->
        <test-method status="PASS" signature="tc001()[pri:0, instance:testing.TC_001@7c0e2abd]" name="tc001" duration-ms="6" started-at="2020-05-08T01:14:35Z" finished-at="2020-05-08T01:14:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc001 -->
        <test-method status="PASS" signature="cleanup()[pri:0, instance:testing.TC_001@7c0e2abd]" name="cleanup" is-config="true" duration-ms="0" started-at="2020-05-08T01:14:35Z" finished-at="2020-05-08T01:14:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanup -->
        <test-method status="PASS" signature="enterURL()[pri:0, instance:testing.TC_001@7c0e2abd]" name="enterURL" is-config="true" duration-ms="0" started-at="2020-05-08T01:14:35Z" finished-at="2020-05-08T01:14:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- enterURL -->
        <test-method status="PASS" signature="tc002()[pri:0, instance:testing.TC_001@7c0e2abd]" name="tc002" duration-ms="1" started-at="2020-05-08T01:14:35Z" finished-at="2020-05-08T01:14:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc002 -->
        <test-method status="PASS" signature="cleanup()[pri:0, instance:testing.TC_001@7c0e2abd]" name="cleanup" is-config="true" duration-ms="0" started-at="2020-05-08T01:14:35Z" finished-at="2020-05-08T01:14:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanup -->
        <test-method status="FAIL" signature="tc003()[pri:0, instance:testing.TC_001@7c0e2abd]" name="tc003" duration-ms="3" started-at="2020-05-08T01:14:35Z" finished-at="2020-05-08T01:14:35Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Assertion Failed expected:<Hello> but was:<Hel1lo>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Assertion Failed expected:<Hello> but was:<Hel1lo>
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at testing.TC_001.tc003(TC_001.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc003 -->
        <test-method status="PASS" signature="enterURL()[pri:0, instance:testing.TC_001@7c0e2abd]" name="enterURL" is-config="true" duration-ms="0" started-at="2020-05-08T01:14:35Z" finished-at="2020-05-08T01:14:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- enterURL -->
        <test-method status="PASS" signature="cleanup()[pri:0, instance:testing.TC_001@7c0e2abd]" name="cleanup" is-config="true" duration-ms="1" started-at="2020-05-08T01:14:35Z" finished-at="2020-05-08T01:14:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanup -->
      </class> <!-- testing.TC_001 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
